db/db.go

```diff
@@ -80,62 +69,45 @@ func Open(dbPath string) (*DB, error) {
 	openMu.Lock()
 	defer openMu.Unlock()
 
-	dbc, err := sql.Open("sqlite3_with_hook", dbPath)
+	os.MkdirAll(filepath.Dir(dbPath), 0755)
```

Comment:
Just to explain my reasoning. The directory is also needed by the Store, but the Store is more fundamental to the system than the SQLite file, hence have the Store create it.